### ISSUE TITLE
Weird symbols in text prompt

### ISSUE THREAD

**Issue by benedicteb** [benedicteb/outcast#9]
Right after page number a weird [] appears

**Comment by PaulMag** [comment]
Yes, that is the newline symbol that appear like that for some reason. It
happened when I changed font to monospace (because the other one was too
difficult to read). We should replace the newlines with spaces via some
algorithm, I guess.
